CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/usgs/pubs-services/compare/pubs-services-1.0.0...master)
+### Added
+  - Ability to bypass security for local development
 ### Changed
-- Source of IPDS data for dissemination step.
+  - Source of IPDS data for dissemination step.
```

Comment:
The two spaces here should fix the codacy issue - https://app.codacy.com/manual/usgs_wma_dev/pubs-services/issues?bid=13642041&filters=W3siaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6WyJDb2RlU3R5bGUiXX1d